build-provider.ps1

```diff
@@ -0,0 +1,56 @@
+<# Build NuGetProvider to be used immediately by PackageManagement #>
+param(
+    # Example: C:\code\OneGet
+    [Parameter(Mandatory=$true)]
+    [string]
+    $OneGetRepositoryRoot,
+
+    [Parameter(Mandatory=$false)]
+    [string]
+    [ValidateSet('net451', 'netcoreapp2.0', 'netstandard1.6', 'all')]
+    $Framework = 'net451',
+
+    [Parameter(Mandatory=$false)]
+    [string]
+    $CopyTo
```

Comment:
>$CopyTo [](start = 4, length = 7)

$Destination or $BuildDropPath?

Reply:
Changing to $Destination

---
In reply to: [154491986](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154491986) [](ancestors = 154491986)